components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
Thank you for pointing out. Fixed in PR: https://github.com/wso2/carbon-event-processing/pull/179